webapp/src/components/messages/cloudMessage.scss

```diff
@@ -0,0 +1,30 @@
+.CloudMessage {
```

Comment:
I assume this file needs the copyright comment banner at the top?

Reply:
I added it, but I really don't know. Not very many scss files have it.